flox/core.py

```diff
@@ -1550,7 +1544,10 @@ def _factorize_multiple(by, expected_groups, any_by_dask, reindex):
             *by_,
             chunks=tuple(chunks.values()),
             meta=np.array((), dtype=np.int64),
-            **kwargs,
```

Comment:
again, I prefer the dict  version

Reply:
Using kwargs breaks type checking. Everything becomes Any then and then you are forced to use isinstance checks.